ygen/codegen.go

```diff
@@ -87,6 +87,18 @@ type GoOpts struct {
 	// produced if the GenerateJSONSchema YANGCodeGenerator field is set to
 	// true.
 	SchemaVarName string
+	// GoyangImportPath specifies the path that should be used in the generated
+	// code for importing the goyang/pkg/yang package. If it is not specified
```

Comment:
The logic for having it in both places was that the `ygen` library can be called from places other than `generator`, so it seemed better to have it here, rather than in `generator`.